README.md

```diff
@@ -191,3 +191,9 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 ## Contributing
 
 Bug reports and pull requests are welcome on GitHub at https://github.com/fastruby/skunk/issues.
+
+## FastRuby.io
+
+![fastruby](https://github.com/fastruby/skunk/raw/master/fastruby-logo.png)
```

Comment:
```suggestion
![FastRuby.io | Rails Upgrade Services](https://github.com/fastruby/skunk/raw/master/fastruby-logo.png)
```

Reply:
@bronzdoc Any thoughts about this? Our brand is "FastRuby.io" or "FastRuby.io | Rails Upgrade Services" -- source: https://fastruby.github.io/styleguide/

Reply:
@etagwerker I like "FastRuby.io | Rails Upgrade Services"